src/main/java/com/github/skjolberg/packing/Packager.java

```diff
@@ -16,6 +18,8 @@
 
 public abstract class Packager {
 
+	static final AtomicBoolean ALWAYS_FALSE = new AtomicBoolean(false);
```

Comment:
So this perhaps not 'on the money' anymore - flavours of BooleanSupplier (always true, deadline, deadline + interrupt) would be perhaps a bit cleaner?

Reply:
right, I changed it in the last commit